Cotton Classing Database File Import Plugin/PublisherDataModel/Model/NDBData.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CottonClassingPlugin.PublisherDataModel
+{
+    public class NDBData : BaseObject
+    {
+        public List<NDBRecord> NDBRecords { get; set; }
+
+        public NDBData()
+        {
+            NDBRecords = new List<NDBRecord>();
+        }
+    }
+
+    //public class NDBRecord : BaseObject
+    public class NDBRecord
+    {
+        public string GINCode { get; set; }
+        public string GINBale { get; set; }
+        public string DateClassed { get; set; }
+        public string ReportingBasis { get; set; }
+        public string ModuleTrailerNumber { get; set; }
+        public string NumberOfBales { get; set; }
+        public string OfficialColorGrade { get; set; }
+        public string StapleLength { get; set; }
+        public string Micronaire { get; set; }
+        public string Strength { get; set; }
+        public string LeafGrade { get; set; }
+        public string ExtraneousMatter { get; set; }
+        public string Remarks { get; set; }
+        public string HVIColorGrade { get; set; }
+        public string ColorQuadrant { get; set; }
+        public string RD { get; set; }
+        public string B { get; set; }
+        public string NonLintContent { get; set; }
+        public string FiberLength { get; set; }
+        public string LengthUniformity { get; set; }
+        public string UplandOrPima { get; set; }
+        public string RecordType { get; set; }
+        public string RecordStatus { get; set; }
+        public string CCCLoan { get; set; }
+    }
+
+    public class FlatFileHelper
+    {
+        public static string ConvertFlatFileToJSON(string input)
```

Comment:
This works, but there are easier ways to do this.   I would recommend passing in the path to the file vs. the file contents, and then within the method use File.ReadAllLines() which will split the lines for you.   Then, instead of hand-coding the json, you could create an NDBRecord class containing string properties matching all these names, and a NDBData class containing a List<NDBRecord> class.   NewtonSoft.Json's JsonConvert.Serialize(ndbData) will then automatically format the json for you.